pkg/tempo/config.go

```diff
@@ -41,11 +41,12 @@ type Config struct {
 
 // RWConfig controls the configuration of exporting to Grafana Cloud
 type RWConfig struct {
```

Comment:
Any reason you didn't change the name of this struct to go with the use of `PushConfig` above? I can't see the struct in use anywhere else.